config/packer/vetafi-ubuntu-14.04.4-server-amd64.json

```diff
@@ -109,6 +127,11 @@
                 "provider": "virtualbox",
                 "version": "0.0.4"
             }
-        }]
+        },
+        {
+            "type": "atlas",
+            "artifact": "qwwqwwq/nodejs-mongodb-redis",
```

Comment:
This isn't a personal folder, this is the name of our image on atlas. When I signed up for atlas I used the username qwwqwwq.